### PR TITLE
[15.0][MIG] product_supplierinfo_barcode

### PR DESCRIPTION
**Pull request by xhnilic3**

### CODE REVIEW

File: product_supplierinfo_barcode/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Lorenzo Battistini @ TAKOBI
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Barcode in supplier pricelist",
+    "summary": "Add a barcode to supplier pricelist items",
+    "version": "15.0.1.0.0",
+    "development_status": "Beta",
```
Comment (rousseldenis) [inline product_supplierinfo_barcode/__manifest__.py:7]: This is the default, not needed

**Review by rousseldenis — APPROVED** [review]
Code review.

@xhnilic3 Could you just take care of meaningful PR's titles ? Thanks

File: product_supplierinfo_barcode/__manifest__.py (hunk 1 above)
Comment (xhnilic3) [reply product_supplierinfo_barcode/__manifest__.py:7]: thank you for the info, will be aware of that next time, sorry

**Comment by xhnilic3** [comment]
@rousseldenis ad meaningful PR's titles, could you tell me more, please? of course it is something I will be very careful about in the future as well, I was following migration wiki, don't know what happened. I am sorry about that

**Comment by rousseldenis** [comment]
> @rousseldenis ad meaningful PR's titles, could you tell me more, please? of course it is something I will be very careful about in the future as well, I was following migration wiki, don't know what happened. I am sorry about that

By default, github try to guess the PR's title (commit message if one, sanitized branch name if several). But this is quite ugly:

image


Change to:

'[MIG] product_supplierinfo_barcode: Migration to 15.0' or 
'[15.0][MIG] product_supplierinfo_barcode'

**Comment by rousseldenis** [comment]
That's nitpicking, but easy reviewers job at first sight in PR's list

**Comment by pedrobaeza** [comment]
Not so nitpicking, because not being able to search in the open pull request by the module name (`product_supplierinfo_barcode`) has provoked in the past some duplicated efforts.

**Comment by xhnilic3** [comment]
@rousseldenis @pedrobaeza thank you for your time and feedback, the title is changed

**Comment by Rad0van** [comment]
@rousseldenis & @pedrobaeza the commit order is not followed here - missing black/isort/prettier. Would that be considered a blocker?

**Comment by rousseldenis** [comment]
> @rousseldenis & @pedrobaeza the commit order is not followed here - missing black/isort/prettier. Would that be considered a blocker?

Nope, that means there is no formatting changes to add here

**Review by Rad0van — APPROVED** [review]
LGTM. Tested.

**Comment by pedrobaeza** [comment]
Well, the setup folder should be added in such pre-commit run

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
> Well, the setup folder should be added in such pre-commit run

But as @pedrobaeza said, I see indeed setup files in commit. They should appear when you run pre-commit like in migration guide.

@xhnilic3 Could you update this accordingly ?

**Pull request merged**